### PR TITLE
Social media links are added at the footer and a heart with pulse

### LINKED ISSUE #6: Add social media links and such
Would be great to add links to our social media places. 

- Facebook Group/Page
- Twitter
- Medium
- GitHub 

and the "Made with <3 by Florida Hackers" footer mark like in the [Floatie](http://floatie.floridahackers.com/) site

### PR DESCRIPTION
**Pull request by raviigarg**
I have added the social media links on footer and a message that says "Made with <3 by Florida Hackers", as you have asked.
screenshot 8

___

Closes #6

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <title>Florida Hackers</title>
         <meta name="description" content="A state-wide community for all students in tech." />
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="/dist/styles.css" type="text/css" />
+        <link rel="stylesheet" href="dist/styles.css" type="text/css" />
```
Comment (jalvarado91) [inline public/index.html]: We shouldn't need to remove the slash or it'll break in prod.

File: public/dist/styles.css [hunk 2]
```diff
@@ -10,4 +10,73 @@
...
....card-wrap .sticky{padding-top:80px}.EventForm{background-color:#fdf2d3;margin-right:36px;border-radius:4px;box-shadow:0 2px 8px 0 rgba(0,0,0,.14),0 1px 4px -1px rgba(0,0,0,.2);padding:24px 32px}.EventForm .InputGroup{margin-bottom:28px}.EventForm .InputGroup label{color:#7b5521;display:block;font-weight:600;margin-bottom:8px}.EventForm .InputGroup input[type=email],.EventForm .InputGroup input[type=text]{border:none;background:none;border-bottom:2px solid #7b5521;display:inline-block;font-size:1em;padding:6px 0 4px}.EventForm .InputGroup ::placeholder{color:rgba(130,83,41,.3);font-weight:100}.EventForm .InputGroup.color input{text-transform:uppercase}.EventForm .InputGroup.color span{color:#7b5521;border-bottom:2px solid #7b5521;font-size:1em;padding:6px 4px 4px 0;display:inline-block}.EventForm .InputGroup .helper-text{display:block;color:rgba(130,83,41,.8);font-size:.9em;margin-top:8px}.EventForm .CheckboxInput{margin-bottom:12px}.EventForm .CheckboxInput .box{width:20px;height:20px;position:relative;margin-right:10px;border-radius:3px;display:inline-block;background:transparent;z-index:1;cursor:pointer;border:2px solid #825329;vertical-align:sub}.EventForm .CheckboxInput .inner
```
Comment (jalvarado91) [inline public/dist/styles.css]: Generally we don't want to modify the dist files, as they get built by Travis. Can you put these changes in one of the SASS files? Inside the `home.scss` file might be a good place.

**Review by jalvarado91 — CHANGES_REQUESTED** [review]

**Comment by raviigarg** [comment]
Can you please tell me how i can add these changes to scss file. I don't have knowledge about scss files.

File: npm-debug.log [hunk 3]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (jalvarado91) [inline npm-debug.log]: Can you remove this file? This shouldn't be in this codebase

File: prepros-6.config [hunk 4]
```diff
@@ -0,0 +1,280 @@
+{
```
Comment (jalvarado91) [inline prepros-6.config]: Remove this file too.

**Review by jalvarado91 — CHANGES_REQUESTED** [review]

**Review by jalvarado91 — APPROVED** [review]
Looks good!

**Pull request merged**